Assets/MRTK/SDK/Features/Input/Handlers/DefaultTransformSmoothingLogic.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    /// <summary>
+    /// A default implementation of TransformSmoother for smoothing transforms.
+    /// </summary>
+    public class DefaultTransformSmoothingLogic : TransformSmoothingLogic
+    {
+        public override Vector3 SmoothPosition(Vector3 source, Vector3 goal, float lerpTime, float deltaTime)
```

Comment:
Validation is failing because I think `TransformSmoothingLogic` needs to be changed to the new interface.